xbmc/input/actions/ActionTranslator.cpp

```diff
@@ -243,6 +243,10 @@ static const std::map<ActionName, ActionID> ActionMappings = {
     // Voice
     {"voicerecognizer", ACTION_VOICE_RECOGNIZE},
 
+    // CEC
+    ("cec_volup", ACTION_CEC_VOLUME_UP),
+    ("cec_voldown", ACTION_CEC_VOLUME_DOWN),
```

Comment:
where does "cec_volup" and "cec_voldown" come from?

Reply:
Nowhere. I'm not sure if I understand your question as that file seems to be the place to define these strings.

Reply:
Resolving cause of no more reply of the devs, so I still have no idea what to change but I guess I need to resolve this before being able to request a new review.